includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -915,6 +915,22 @@ private static function settings_appearance() {
 
 		WPBDP_Admin_Education::add_tip_in_settings( 'table', 'themes' );
 
+        if ( ! WPBDP_App_Helper::is_font_awesome_enabled() ) {
+			wpbdp_register_setting(
+				array(
+					'id'    => 'fontawesome-enabled',
+					'desc'  => wp_kses_post(
+						sprintf(
+							'<span>' . __( 'Install %s for Font Awesome styles?', 'business-directory-plugin' ) . '</span>',
```

Comment:
Let's use a message that's a bit more descriptive like this:
Use icons in your directory listings and custom fields: Install %s

Reply:
This is much better, thanks :)